modules/installation-uninstall-aws.adoc

```diff
@@ -7,27 +7,87 @@
 
 You can remove a cluster that you installed on Amazon Web Services (AWS).
 
+.Prerequisites
+
+* Deploy an {product-title} cluster.
+* Have a copy of the installation program that you used to deploy the cluster.
+
 .Procedure
 
-. Optionally, from the computer that you used to install the cluster, run the
-following command and record the UUID that it outputs:
+Perform the following steps on the computer that you used to install the cluster.
+
+. Obtain the cluster UUID:
```

Comment:
I don't think I need to do this step if I have the `metadata.json` file. Based on these steps, I only need to cluster UUID to generate the `metadata.json`, which I already have. (I think....)

Reply:
Amended: I didn't see this: _ you can also use this UUID to locate them and remove them_. 
How do I use the UUID to locate services?

Reply:
> I don't think I need to do this step if I have the `metadata.json` file. Based on these steps, I only need to cluster UUID to generate the `metadata.json`, which I already have. (I think....)

Totally agree. The procedure should be like this:
1. Locate the metadata.json file that describes your cluster. This file is in the installation_directory directory that you specified when you deployed your cluster.
2. If you cannot locate the metadata.json file, recreate it; If you can locate the metadata.json file, skip this step
a. Obtain the cluster UUID:
b. Store the cluster name, AWS region, and UUID as environment variables:
c. Generate the metadata.json file for your cluster:
3. Change to the directory that contains the OpenShift Container Platform installation program and run the following command:
4. Optionally, delete the installation_directory directory and the OpenShift Container Platform installation program.

Reply:
@mburke5678, there's no good process for using the UUID to track down the leftover components in AWS. I picked it up from [the beta training docs](https://github.com/openshift/training/blob/master/docs/06-cleanup.md#deleting-the-cluster-and-cleaning-up), but without the process, it's not helpful to point out.

I'm reordering the steps. Thanks!